src/core_plugins/input_control_vis/public/components/vis/__snapshots__/input_control_vis.test.js.snap

```diff
@@ -500,7 +544,11 @@ exports[`Renders range control 1`] = `
           onClick={[Function]}
           type="button"
         >
-          Apply changes
+          <FormattedMessage
+            defaultMessage="Apply changes"
+            id="inputControl.vis.inputControlVis.applyChangesButtonLabel"
+            values={Object {}}
+          />
         </EuiButton>
```

Comment:
The `values={Object {}}` props, which are all over this file, are not needed, right?

Reply:
This a snapshot file, it's generated automatically

Reply:
oh haha 😅 thanks